src/snowflake/snowpark/dataframe.py

```diff
@@ -2192,7 +2193,12 @@ def pivot(
         """
 
         if _emit_ast:
```

Comment:
before this if, there should be something like `stmt = None`. Else, for `_emit_ast=False` the return statement will raise an undefined name exception.

Reply:
Done.